modules/compliance-profiles.adoc

```diff
@@ -59,92 +55,88 @@ $ oc get -n openshift-compliance -oyaml profiles.compliance rhcos4-e8
+    compliance.openshift.io/product-type: Node
+  creationTimestamp: "2022-10-19T12:06:49Z"
+  generation: 1
+  labels:
+    compliance.openshift.io/profile-bundle: rhcos4
+  name: rhcos4-e8
+  namespace: openshift-compliance
+  ownerReferences:
+  - apiVersion: compliance.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ProfileBundle
+    name: rhcos4
+    uid: 22350850-af4a-4f5c-9a42-5e7b68b82d7d
+  resourceVersion: "43699"
+  uid: 86353f70-28f7-40b4-bf0e-6289ec33675b
+rules:
+- rhcos4-accounts-no-uid-except-zero
+- rhcos4-audit-rules-dac-modification-chmod
+- rhcos4-audit-rules-dac-modification-chown
+- rhcos4-audit-rules-execution-chcon
+- rhcos4-audit-rules-execution-restorecon
+- rhcos4-audit-rules-execution-semanage
+- rhcos4-audit-rules-execution-setfiles
+- rhcos4-audit-rules-execution-setsebool
+- rhcos4-audit-rules-execution-seunshare
+- rhcos4-audit-rules-kernel-module-loading-delete
+- rhcos4-audit-rules-kernel-module-loading-finit
+- rhcos4-audit-rules-kernel-module-loading-init
+- rhcos4-audit-rules-login-events
+- rhcos4-audit-rules-login-events-faillock
+- rhcos4-audit-rules-login-events-lastlog
+- rhcos4-audit-rules-login-events-tallylog
+- rhcos4-audit-rules-networkconfig-modification
+- rhcos4-audit-rules-sysadmin-actions
+- rhcos4-audit-rules-time-adjtimex
+- rhcos4-audit-rules-time-clock-settime
+- rhcos4-audit-rules-time-settimeofday
+- rhcos4-audit-rules-time-stime
+- rhcos4-audit-rules-time-watch-localtime
+- rhcos4-audit-rules-usergroup-modification
+- rhcos4-auditd-data-retention-flush
+- rhcos4-auditd-freq
+- rhcos4-auditd-local-events
+- rhcos4-auditd-log-format
+- rhcos4-auditd-name-format
+- rhcos4-auditd-write-logs
+- rhcos4-configure-crypto-policy
+- rhcos4-configure-ssh-crypto-policy
+- rhcos4-no-empty-passwords
+- rhcos4-selinux-policytype
+- rhcos4-selinux-state
+- rhcos4-service-auditd-enabled
+- rhcos4-sshd-disable-empty-passwords
+- rhcos4-sshd-disable-gssapi-auth
+- rhcos4-sshd-disable-rhosts
+- rhcos4-sshd-disable-root-login
+- rhcos4-sshd-disable-user-known-hosts
+- rhcos4-sshd-do-not-permit-user-env
+- rhcos4-sshd-enable-strictmodes
+- rhcos4-sshd-print-last-log
+- rhcos4-sshd-set-loglevel-info
+- rhcos4-sysctl-kernel-dmesg-restrict
+- rhcos4-sysctl-kernel-kptr-restrict
+- rhcos4-sysctl-kernel-randomize-va-space
+- rhcos4-sysctl-kernel-unprivileged-bpf-disabled
+- rhcos4-sysctl-kernel-yama-ptrace-scope
+- rhcos4-sysctl-net-core-bpf-jit-harden
+title: Australian Cyber Security Centre (ACSC) Essential Eight
 ----
 
-* View the rules within a desired profile:
-+
-[source,terminal]
-----
-$ oc get -n <namespace> -oyaml rules.compliance <rule_name>
-----
-+
-This example displays the `rhcos4-audit-rules-login-events` rule in the `rhcos4` profile:
+* View the rules within a desired profile. This example displays the `rhcos4-audit-rules-login-events` rule in the `rhcos4` profile:
 +
 [source,terminal]
 ----
```

Comment:
Better to update to
$ oc get -n openshift-compliance -oyaml rules rhcos4-audit-rules-login-events